index.html

```diff
@@ -1,8 +1,9 @@
 <html>
     <head>
         <title>World greeter</title>
+	<script src="index.js"></script>
```

Comment:
I think your script tag here needs a defer keyword. Currently, the script is loaded before the DOM.